onnxruntime/core/providers/partitioning_utils.h

```diff
@@ -54,6 +54,7 @@ Create the supported partitions for the execution provider.
 @param on_group_closed_fn Callback to indicate a completed partition node group.
 @param generate_metadef_name_fn Callback to create the name for the MetaDef.
 @param execution_provider_name Name of execution provider creating the ComputeCapability instance.
+@param current_ep ExecutionProviderType of the EP creating this ComputeCapability instance.
```

Comment:
>current_ep [](http://example.com/codeflow?start=7&length=10)

here and elsewhere, how about "execution_provider_type" instead of "current_ep"?